doc/README.rst

```diff
@@ -59,56 +67,11 @@ index via the command line
 
 .. note::
 
- The command ``helpdesk()`` needs to know which browser to launch in order to display
- the help pages. The browser is set as an option of helpdesk. Please see the file
- ``~/.nestrc`` for an example setting firefox as browser.
- Please note that the command helpdesk does not work if you have compiled
- NEST with MPI support, but you have to enter the address of the helpdesk
- (file:///install/path/share/doc/nest/index.html) manually into the browser.
- Please replace ``/install/path`` with the path under which NEST is installed.
-
-Local install of user documentation
---------------------------------------
-
-If you are using Linux or macOS and want to locally install the docs
-and generate the html files, follow the steps below.
-
-Requirements
-~~~~~~~~~~~~
-
-To keep everything simple and easy to maintain, the installation with
-conda is described here. So all you need is conda. If you don’t have it
-yet, `miniconda <https://conda.io/miniconda.html>`__ is recommended.
-
-Installation
-~~~~~~~~~~~~
-
-With the following steps a full development environment is installed -
-even the newest NEST release.
-
-Change to the doc/ folder in the source directory (if you’re not already there).
-
-::
-
-   cd ./doc
-
-Create and activate the environment.
-
-::
-
-   conda env create -f nest_doc_conda_env.yml
-   conda update -n base conda
-   conda activate nest_doc
-
-Now generate the html files. They are then located in ./docs/_build/html.
-
-::
-
-   make html
-
-If you want to deactivate and/or delete the build environment:
-
-::
-
-   conda deactivate
-   conda remove --name nest_doc --all
+ The command ``helpdesk()`` needs to know which browser to launch in
+ order to display the help pages. The browser is set as an option of
+ helpdesk. Please see the file ``~/.nestrc`` for an example setting
+ firefox as browser.  Please note that the command helpdesk does not
+ work if you have compiled NEST with MPI support, but you have to
+ enter the address of the helpdesk
+ (file://<nest_install_dir>/share/doc/nest/index.html) manually into
```

Comment:
This is for the overall helpdesk. How does one get from there to SLI? The SLI command index is in `<install_dir>/share/doc/nest/help/helpindex.html`-

Reply:
With the present changes, the SLI helpdesk is actually [not installed anymore](https://github.com/nest/nest-simulator/pull/1843/files/ee754d1254d32a64aeae4c90427ae973bb265ed2#diff-652f011e4f45934b3684d4596670e3847750ec3289b5092d4f8847b1ca6a75edL62-L64). Stay tuned for a follow-up PR, refactoring the build and install for that.